2200021/1225.py

```diff
@@ -0,0 +1,20 @@
+# 1225 암호생성기
+
+from collections import deque
+
+for _ in range(10):
+    t = int(input())
+    q = deque(list(map(int,input().split())))
```

Comment:
덱 활용 아주 좋습니다! 💯 
```suggestion
    q = deque(map(int,input().split()))
```
바로 `deque`을 적용해주셔도 괜찮습니다 😄